cmd/sonobuoy/app/modes.go

```diff
@@ -48,6 +48,10 @@ const (
 	nonDisruptiveSkipList = `\[Disruptive\]|NoExecuteTaintManager`
 	conformanceFocus      = `\[Conformance\]`
 	quickFocus            = "Pods should be submitted and removed"
+
+	E2eModeNetworking string = "conformance+networking"
+	networkingFocus   string = `\[Conformance\]|NetworkPolicy|sig-network|StatefulSet`
+	networkingSkip    string = `\[Disruptive\]`
```

Comment:
I wonder if we should we also skip `Storage` since that's often associated w/ StatefulSet related stuff...